examples/fabric-bridge-app/linux/README.md

```
@@ -100,13 +100,13 @@ defined:
     Pull Docker Images
 
     ```
-    docker pull ghcr.io/project-chip/chip-build-crosscompile:200
+    docker pull ghcr.io/project-chip/chip-build-crosscompile:211
     ```
 
     Run docker
 
     ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)